optlang/symbolics.py

```diff
@@ -84,12 +86,14 @@ def __getnewargs__(self):
     def add(*args):
         if len(args) == 1:
             args = args[0]
-        return sum(args)
+        return Add(*args)
 
     def mul(*args):
         if len(args) == 1:
             args = args[0]
-        return reduce(operator.mul, args, 1)
+        if len(args) == 0:
+            return(Zero)  # if you multiply nothing the result should be zero
+        return reduce(operator.mul, args, One)
```

Comment:
This is slower than `Mul(*args)` in symengine

Reply:
Thanks for the heads up! :)